templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -284,7 +284,7 @@
     <!-- end of soft credit -->
 
     <!-- start of PCP -->
-    {if $siteHasPCPs && !$payNow}
+    {if array_key_exists($form, 'pcp_made_through_id') && !$payNow}
```

Comment:
This looks backwards: `array_key_exists('pcp_made_through_id', $form)`

Reply:
But also if $suffix isn't blank this wouldn't be true, right?

Reply:
Hmm but the tpl assumes it always is blank. Civi is weird.

Reply:
Yep - agree on all counts :-)